tests/ui/or_fun_call.stderr

```diff
@@ -72,17 +72,11 @@ error: use of `unwrap_or` followed by a function call
 LL |     let _ = stringy.unwrap_or("".to_owned());
    |                     ^^^^^^^^^^^^^^^^^^^^^^^^ help: try this: `unwrap_or_else(|| "".to_owned())`
 
-error: use of `ok_or` followed by a function call
-  --> $DIR/or_fun_call.rs:74:17
-   |
-LL |     let _ = opt.ok_or(format!("{} world.", hello));
-   |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: try this: `ok_or_else(|| format!("{} world.", hello))`
```

Comment:
`format!` now expands to a block. While format is explicitly handled in `expect_fun_call`, it is not in `or_fun_call`.